packages/core/src/capabilities/NodeInfo.ts

```diff
@@ -252,21 +282,30 @@ export function parseNodeProtocolInfo(
 		default:
 			isFrequentListening = false;
 	}
-	const supportsBeaming = !!(capability & 0b0001_0000);
+	const supportsBeaming = (!isLongRange) && !!(capability & 0b0001_0000);
 
 	let nodeType: NodeType;
-	switch (capability & 0b1010) {
-		case 0b1000:
+
+	switch (
+		isLongRange
+			? (0b1_0000_0000 | (capability & 0b0010))
+			: (capability && 0b1010)
+	) {
+		case 0b0_0000_1000:
+		case 0b1_0000_0000:
 			nodeType = NodeType["End Node"];
 			break;
-		case 0b0010:
+		case 0b0_0000_0010:
+		case 0b1_0000_0010:
 		default:
+			// BUGBUG: is Controller correct for default && isLongRange?
 			nodeType = NodeType.Controller;
 			break;
 	}
 
-	const hasSpecificDeviceClass = !!(capability & 0b100);
-	const supportsSecurity = !!(capability & 0b1);
+	const hasSpecificDeviceClass = isLongRange || !!(capability & 0b100);
+	// BUGBUG: can we assume security is true?
+	const supportsSecurity = isLongRange || !!(capability & 0b1);
```

Comment:
Given that I couldn't include a node except with S2 security, and they don't define the bit for LR versions of this struct, I suspect the answer is "security is always available if LR", but I couldn't find that in the spec.